cvtoolkit/datasets/yolo_labels_dataset.py

```diff
@@ -57,6 +67,9 @@ def from_yolo_validation_json(
                 logger.error("Unknown json content, aborting.")
                 return None
 
+        if not confidence_threshold:
```

Comment:
If you're setting up the confidence_threshold by default to 0.0, would it not be more transparent to do that as a default value of the function (instead of None and then changing it)?